src/sentry/tsdb/redis.py

```diff
@@ -110,19 +110,28 @@ def validate(self):
             label='TSDB',
         )
 
-    def make_key(self, model, rollup, timestamp, key):
+    def add_environment_parameter(self, key, environment):
+        if environment is not None:
+            return '{}?e={}'.format(key, environment)
```

Comment:
Note that this doesn't encode the environment part in any way, so is not future-proof if we'd like to add additional parameters to keys later.

Reply:
Also, encoding this as a base-10 textual representation of an integer is kind of a wasteful representation. We could choose a more efficient format if we're concerned about that kind of thing.